CHANGELOG.md

```diff
@@ -54,6 +54,9 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 - The `remote status` command will now print the username, realname, and email
   of the logged-in user, if available.
+- Add monitoring feature support, which requires the usage of an additional tool.
+  This tool will put apptainer starter into a newly created cgroup and collect cgroup's
+  system metrics.
```

Comment:
Don't you want to mention the tool's name, so people will search for it if they're interested?